src/ml_tooling/storage/file.py

```diff
@@ -0,0 +1,106 @@
+from ml_tooling.storage.base import Storage, StorageEnvironment
+from ml_tooling.utils import MLToolingError
+
+from typing import List, Any
+from contextlib import contextmanager
+from pathlib import Path
+from joblib import load, dump
+
+
+class FileStorage(Storage):
+    
+    def __init__(self, dir_path=None):
+        self.dir_path = Path.cwd() if dir_path is None else Path(dir_path)
+        self.filename = None
+        self.file = None
+
+    def __enter__(self):
+        if not self.dir_path.exists():
+            self.dir_path.mkdir(parents=True)
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        return False
+
+    def get_list(self) -> List[Path]:
+        """
+        Finds a list of estimator filenames in the FileStorage directory,
+        if the path given is to a file, the directory in which the file resides
+        is used to find the list.
+
+        Parameters
+        ----------
+        None
+
+        Example
+        -------
+        Find and return estimator paths in a given directory:
+            my_estimators = FileStorage('path/to/dir').get_list()
+
+        Returns
+        -------
+        List[Path]
+            list of paths to files
+        """
+        return sorted(self.dir_path.glob('*.pkl'))
+    
+    def load(self, file_path) -> Any:
+        """
+        Loads a joblib pickled estimator from given filepath and returns a Model
+        instatiated with the estimator.
+
+        Parameters
+        ----------
+        file_path: str
+            Path to estimator pickle file
+
+        Example
+        -------
+        We can load a saved pickled estimator from disk directly from FileStorage:
+            storage = FileStorage()
+            my_estimator = storage.load('path/to/estimator')
+
+        We now have a trained estimator loaded.
+
+        Returns
+        -------
+        Object
+            any python object loaded from disk
+        """
+        estimator_path = Path(file_path)
+        return load(estimator_path)
```

Comment:
it might be more readable if is is joblib.load

Reply:
got it